src/main/java/com/cst438/domain/GradeRepository.java

```diff
@@ -2,11 +2,19 @@
 
 import org.springframework.data.jpa.repository.Query;
 import org.springframework.data.repository.CrudRepository;
+import java.util.List;
 
 public interface GradeRepository extends CrudRepository<Grade, Integer> {
 
     // TODO uncomment the following lines as needed
 
-//    @Query("select g from Grade g where g.assignment.assignmentId=:assignmentId and g.enrollment.enrollmentId=:enrollmentId")
-//    Grade findByEnrollmentIdAndAssignmentId(int enrollmentId, int assignmentId);
+   @Query("select g from Grade g where g.assignment.assignmentId=:assignmentId and g.enrollment.enrollmentId=:enrollmentId")
+   Grade findByEnrollmentIdAndAssignmentId(int enrollmentId, int assignmentId);
+
+   @Query("select g from Grade g where g.assignment.assignmentId=:assignmentId")
+   List<Grade> findByAssignmentId(int assignmentId);
+
+   @Query("select g from Grade g where g.enrollment.enrollmentId=:enrollmentId")
```

Comment:
Added this line because of the networking issues I had at work. Let me know if this is actually needed.